net-misc/ssh-askpass-fullscreen/ssh-askpass-fullscreen-1.2-r1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="A small SSH Askpass replacement written with GTK2"
+HOMEPAGE="https://github.com/atj/ssh-askpass-fullscreen"
+SRC_URI="https://github.com/atj/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~riscv ~sparc ~x86"
+
+RDEPEND=">=x11-libs/gtk+-2.10.0:2
```

Comment:
```suggestion
RDEPEND="
	>=x11-libs/gtk+-2.10.0:2
```

Reply:
this has no maintainer, so we don't need a best practice style-go-ahead from anyone else.